ip/event_manager.go

```diff
@@ -220,9 +220,9 @@ func (m *manager) sendHealthcheckResults(ctx context.Context, healthy bool) {
 		m.sendEvent(HealthCheckSuccessEvent)
 	} else {
 		m.failingCount++
-		log.WithField("failing_count", m.failingCount).Info("healthcheck failed (retry)")
+		log.WithField("failing_count", m.failingCount).Infof("healthcheck failed (retry): %v", err)
```

Comment:
Same as the review  here (https://github.com/Scalingo/sand/pull/40#pullrequestreview-314075336), maybe use `log.WithError` to have the error in a parseable field

Reply:
ack